jssrc/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = function ttf2woff2(inputContent) {
 
   // Retrieve output
   outputSize = theTTFToWOFF2Module.getValue(outputSizePtr, 'i32');
-  outputContent = Buffer.from(outputSize);
+  outputContent = Buffer.allocUnsafe(outputSize);
```

Comment:
Considering the fact the font may be publicly available, I think that a `alloc` instead of a `allocUnsafe` worth the perfomance drop to avoid potential leaks.